app/helpers/policy_disregards_helper.rb

```diff
@@ -0,0 +1,78 @@
+module PolicyDisregardsHelper
+  include CheckAnswersHelper
+
+  ATTRIBUTES = %i[
+    england_infected_blood_support
+    vaccine_damage_payments
+    variant_creutzfeldt_jakob_disease
+    criminal_injuries_compensation_scheme
+    national_emergencies_trust
+    we_love_manchester_emergency_fund
+    london_emergencies_trust
+  ].freeze
+
+  def policy_disregards_hash(policy_disregards)
+    items = policy_disregards_as_array(policy_disregards)
+    items&.compact!
+    return nil unless items.present?
```

Comment:
`return nil unless items.present?` 
can be
`return unless items.present?` which returns nil without needing to specify it